sqlcipher_flutter_libs/README.md

```diff
@@ -44,8 +44,9 @@ void main() {
 Depending on your target platform, additional dependencies may be needed:
 
 - Android: Uses a precompiled library, no additional setup is needed.
+  - If you want to access SQLite Cipher from the native Android project, make sure to check the [migration guide](https://www.zetetic.net/sqlcipher/sqlcipher-for-android-migration/) from `net.zetetic:android-database-sqlcipher` to `net.zetetic:sqlcipher-android`.
```

Comment:
I'm not sure if we should mention this at all. The dependency we add doesn't add it to the compile classpath of the main app project, and if developers add the dependency to their main `build.gradle` they should know what they're doing.

Maybe we should bump the version to `0.6.0` and mention this in the changelog instead?